src/librustc/mir/interpret/error.rs

```diff
@@ -309,7 +319,8 @@ impl<'tcx, O> EvalErrorKind<'tcx, O> {
             ReadForeignStatic =>
                 "tried to read from foreign (extern) static",
             InvalidPointerMath =>
-                "attempted to do invalid arithmetic on pointers that would leak base addresses, e.g. comparing pointers into different allocations",
+                "attempted to do invalid arithmetic on pointers that would leak base addresses,
+                e.g. comparing pointers into different allocations",
```

Comment:
This needs a backslash to not change the actual string being computed:
```
                "attempted to do invalid arithmetic on pointers that would leak base addresses, \
                e.g. comparing pointers into different allocations",
```
Here and elsewhere.